### PR TITLE
add filter features using qza

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: .github/workflows/qiita-plugin-ci.yml [hunk 1]
```diff
@@ -62,8 +62,9 @@ jobs:
 
           # Setting up main qiita conda environment
           conda config --add channels conda-forge
-          conda create -q --yes -n qiita python=3.6 pip libgfortran numpy nginx cython redis
+          conda create -q --yes -n qiita python=3.6 pip==9.0.3 libgfortran numpy nginx cython redis
```
Comment (wasade) [inline .github/workflows/qiita-plugin-ci.yml:65]: why pin pip?

**Review by wasade — COMMENTED** [review]
just a brief question

File: .github/workflows/qiita-plugin-ci.yml (hunk 1 above)
Comment (antgonza) [reply .github/workflows/qiita-plugin-ci.yml:65]: It the "simplest" solution I have found [to this error](https://github.com/qiita-spots/qp-qiime2/runs/4245456034?check_suite_focus=true#step:5:1436); basically pin pip so conda can do it's thing and then update.

Comment (antgonza) [reply .github/workflows/qiita-plugin-ci.yml:65]: It the "simplest" solution I have found [to this error](https://github.com/qiita-spots/qp-qiime2/runs/4245456034?check_suite_focus=true#step:5:1436); basically pin pip so conda can do its thing and then update.

Comment (antgonza) [reply .github/workflows/qiita-plugin-ci.yml:65]: It the "simplest" solution I have found [to this error](https://github.com/qiita-spots/qp-qiime2/runs/4245456034?check_suite_focus=true#step:5:1436); basically pin pip so conda can do its thing and then update.

Comment (wasade) [reply .github/workflows/qiita-plugin-ci.yml:65]: that's frustrating, hope they fix soon

**Comment by antgonza** [comment]
Thank you @wasade

**Pull request merged**